backend/controllers/event.controller.ts

```diff
@@ -328,6 +328,82 @@ export const getEventById = async (
   }
 };
 
+// Get Events by User Id. Sends three json objects
+export const getEventByUserId = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    // Validate request id param
+    const userId = IdParamSchema.parse(req.params).id;
+
+    // Get event from db
+    const eventsCreatedByUser = await prisma.event.findMany({
+      where: {
+        id: userId,
+      },
+    });
+
+    const eventsInterested = await prisma.event.findMany({
```

Comment:
One thing I just thought of was what if they click both? Cause right now on the front-end they can heart it (to say they are interested) but also click 'going.' Should we limit it to one or the other... or both

We might have to think of the best way to approach this:
@jcharlpretorius @jtran217 @JamesRobert20

Reply:
The heart makes it simple for users to get notified of the event details. In my opinion, I would stick with the heart, to make the experience simple for the user. I can always change it up if the team sees it differently.

Reply:
I think it makes sense to limit it to a single response per event. 

Right now the db model is set up so a UserEventResponse record is uniquely identified by the combination of UserId and EventId. Changing it would be trivial if we wanted to, but I don't think it makes sense to